### PR TITLE
1 byte donation for Fabrizio ;)

### PR DESCRIPTION
**Pull request by IrgendwerA8**
Only unit tested, but hey, what could go wrong? ;)

### CODE REVIEW

File: libsrc/vic20/cgetc.s [hunk 1]
```diff
@@ -40,8 +40,8 @@ L3:     jsr     KBDREAD         ; Read char and return in A
         bne     seton           ; Go set it on
         lda     CURS_FLAG       ; Is the cursor currently off?
         bne     crs9            ; Jump if yes
-        lda     #1
-        sta     CURS_FLAG       ; Mark it as off
+        inx                     ; X is now 1 
+        stx     CURS_FLAG       ; Mark it as off
```
Comment (greg-king5) [inline libsrc/vic20/cgetc.s]: Just increment `CURS_FLAG`!

Comment (IrgendwerA8) [reply libsrc/vic20/cgetc.s]: Of course! Thanks for the review!

**Comment by IrgendwerA8** [comment]
Please have a look into the C16 cgetc() too. Thanks!

**Comment by IrgendwerA8** [comment]
Done. There are just too many 8 bit cbm targets for an Atarian... ;)

**Pull request merged**